tests/compilation_tests/grid_test.py

```diff
@@ -98,8 +101,20 @@ def test_node_boundaries(self):
         y_range = get_vincenty_y((lat_min, long_min), (lat_max, long_max))
         x_range = get_vincenty_x((lat_min, long_min), (lat_max, long_max))
         top_right_node = g.nodes[g.get_num_rows()-1][g.get_num_cols()-1]
-        self.assertLessEqual(top_right_node.get_m_coords()[0], x_range, "The meters grid shouldn't be larger than the lat bounds")
-        self.assertLessEqual(top_right_node.get_m_coords()[1], y_range, "The meters grid shouldn't be larger than the long bounds")
+        self.assertLessEqual(top_right_node.get_m_coords()[
+                             0], x_range, "The meters grid shouldn't be larger than the lat bounds")
+        self.assertLessEqual(top_right_node.get_m_coords()[
+                             1], y_range, "The meters grid shouldn't be larger than the long bounds")
+
+
+class TestInside(unittest.TestCase):
+    def test_inside(self):
```

Comment:
Could we add a case for a point that is on the edge of a triangle? 🤔

Reply:
Also something I just realized 🤔 `isInsideTriangle` is in camel case while the rest of our code is in snake case. As a part of a ~realization~ resulting from this test, I think it would be good to fix this!

Reply:
Godd jobb so I think that a test for the circles  that check the extremes also makes sense